packages/react/src/components/DropdownSelector/DropdownMenuItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import './index.css'
+
+import MenuItem, { MenuItemProps } from '../MenuItem'
+import { MenuListContext } from '../MenuList/MenuListContext'
+import { useContext } from 'react'
+import Icon from '../../Icon'
+
+export type DropdownMenuItemProps = Omit<MenuItemProps, 'as'>
+
+/**
+ * DropdownSelectorの選択肢として使うMenuItem
+ */
+export default function DropdownMenuItem(props: DropdownMenuItemProps) {
+  const { value: ctxValue } = useContext(MenuListContext)
+  const isSelected = props.value === ctxValue
+  const { children, ...rest } = props
+
+  return (
+    <MenuItem {...rest} aria-selected={isSelected}>
+      {isSelected && (
+        <Icon
+          className="charcoal-dropdown-selector-menu-item-icon"
+          name="16/Check"
+        />
+      )}
+      <span
+        className="charcoal-dropdown-selector-menu-item"
+        aria-selected={isSelected}
```

Comment:
`aria-selected` is used for grid-cell, option, row, and tab roles, so `data-selected` is preferred.

https://www.osaka-kyoiku.ac.jp/~joho/html5_ref/wai-aria/states_and_properties.html#aria-selected